### ISSUE TITLE
Gold score

### ISSUE THREAD

**Issue by hytseng0509** [sangwoomo/gold#1]
Hi,

Thanks for sharing the source code!
According to the implementation [here](https://github.com/sangwoomo/gold/blob/0feb18a78a2c56001dbbd8eabf807bbb409d25db/utils/utils.py#L63), it seems that you directly use the logits predicted from the discriminator as the weight for the re-weighting training. Does it mean that some weights might be negative (thus reversing the direction of the gradient computed from the re-weighted loss [here](https://github.com/sangwoomo/gold/blob/0feb18a78a2c56001dbbd8eabf807bbb409d25db/train/train_acgan_semi.py#L82))?

**Comment by sangwoomo** [comment]
Hi,

Your understanding is exactly right.
My intuition was to suppress the _over_-generated regions.
However, I did not investigated much variations of the loss; hence, one would find the better one :)

**Comment by hytseng0509** [comment]
Thanks for the reply!
